iis-workflows/iis-workflows-export-actionmanager/src/main/resources/eu/dnetlib/iis/workflows/export/actionmanager/sequencefile/oozie_app/workflow.xml

```diff
@@ -1,652 +1,715 @@
+        </property>
+    </parameters>
+
+    <global>
+        <job-tracker>${jobTracker}</job-tracker>
+        <name-node>${nameNode}</name-node>
+        <configuration>
+            <property>
+                <name>mapred.job.queue.name</name>
+                <value>${queueName}</value>
+            </property>
+            <property>
+                <name>mapreduce.inputformat.class</name>
+                <value>org.apache.avro.mapreduce.AvroKeyInputFormat</value>
+            </property>
+            <property>
+                <name>mapreduce.outputformat.class</name>
+                <value>org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat</value>
+            </property>
+            <property>
+                <name>mapred.mapoutput.key.class</name>
+                <value>org.apache.hadoop.io.Text</value>
+            </property>
+            <property>
+                <name>mapred.mapoutput.value.class</name>
+                <value>org.apache.hadoop.io.Text</value>
+            </property>
+            <property>
+                <name>mapred.output.key.class</name>
+                <value>org.apache.hadoop.io.Text</value>
+            </property>
+            <property>
+                <name>mapred.output.value.class</name>
+                <value>org.apache.hadoop.io.Text</value>
+            </property>
+
+            <property>
+                <name>mapred.reduce.tasks.speculative.execution</name>
+                <value>false</value>
+            </property>
+            <property>
+                <name>mapred.map.tasks.speculative.execution</name>
+                <value>false</value>
+            </property>
+            <property>
+                <name>mapreduce.map.speculative</name>
+                <value>false</value>
+            </property>
+            <property>
+                <name>mapreduce.reduce.speculative</name>
+                <value>false</value>
+            </property>
+
+            <property>
+                <name>mapred.compress.map.output</name>
+                <value>true</value>
+            </property>
+            <property>
+                <name>mapred.output.compress</name>
+                <value>true</value>
+            </property>
+            <property>
+                <name>mapred.output.compression.type</name>
+                <value>BLOCK</value>
+            </property>
+
+            <property>
+                <name>mapred.reduce.tasks</name>
+                <value>0</value>
+            </property>
+            <property>
+                <name>io.serializations</name>
+                <value>org.apache.hadoop.io.serializer.WritableSerialization,org.apache.hadoop.io.serializer.avro.AvroSpecificSerialization,org.apache.hadoop.io.serializer.avro.AvroReflectSerialization,org.apache.avro.hadoop.io.AvroSerialization
+                </value>
```

Comment:
Since the commit fixes indentation, I'll complain here. This closing tag should not be on a separate line.